test/Microsoft.AspNetCore.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs

```diff
@@ -386,21 +386,21 @@ public void CanIdentifyWhiteSpace_WithoutIgnoringStrings()
             tagHelperContent.Append("Hello");
 
             // Assert
-            Assert.False(tagHelperContent.IsWhiteSpace);
+            Assert.False(tagHelperContent.IsEmptyOrWhiteSpace);
         }
 
         [Fact]
-        public void IsEmpty_InitiallyTrue()
+        public void IsEmptyOrWhiteSpace_InitiallyTrue()
```

Comment:
My previous [comment](https://github.com/aspnet/Razor/pull/728/files#r58143306) about "now looking for something different" do not apply when the test is specifically about the property.